### DISCUSSION TITLE
Why the Funding section was deleted?

### DISCUSSION THREAD

**Discussion by lamualfa** [tw-in-js/twind#126] [Q&A] [answered]
I think it is okay to open a funding session from Open Collective or other to meet the existing needs.

There are many things we can do using the funds that we get. Examples are like:

- Finance the Mainteners
- Create independent documentation such as Tailwind Docs
- Rent a server to create an independent Playground
- etc

Funding sessions are not a bad thing. although I am not a maintener, I hope the funding session can be reopened. Only from that point a layman like me can help the great projects like this.

**Comment by sastan** [comment, marked as answer]
I only wanted to try something. But if you like it – it will come back.

**Reply by sastan** [reply]
Back it is.

**Reply by sastan** [reply]
@lamualfa Thanks a lot for the first ever Twind contribution!!! That means a lot. ❤️ You are awesome!
